JetNews/app/src/main/java/com/example/jetnews/ui/theme/Theme.kt

```diff
@@ -39,9 +42,18 @@ private val DarkThemeColors = darkColors(
     onPrimary = Color.Black,
     secondary = Red300,
     onSecondary = Color.Black,
-    error = Red200
+    error = Red200,
+    onBackground = Color.White
 )
 
+val Colors.articleIconBackground: Color
```

Comment:
@JoseAlcerreca @nickbutcher Do we commonly add extension colors to MaterialTheme or just add them as separate variables?

Reply:
No, we never add extension colors.

These colors should come from the theme. Why are they different? Is there a spec?

Reply:
Since these two colors are not considered within the Figma design Color Scheme I was unsure if I should place them with all the other colors, therefore, I looked in the other Jetpack compose examples to see how you were handling these declarations, I found the extension colors in the Jetsurvey theme and that is why I declared them this way. But I can make the changes, should I declare them as regular variables?

Reply:
My other option would be to add the icon as a png drawable instead of a vector asset. Let me know which approach should I take.

Reply:
My question still applies, why are they different from the Material Theme? isn't it the same red? What spec are you looking at?

Reply:
I added the two colors  to create the Article header icon from a vector asset. 
![image](https://user-images.githubusercontent.com/16503982/116409798-ce040f80-a7f9-11eb-851b-03e8cea84d5a.png)

None of them where considered in the Color Scheme because they were not actually a part of the theme, they were just the color for the icon and the background.
![image](https://user-images.githubusercontent.com/16503982/116410234-2e934c80-a7fa-11eb-8e57-e314ef5e9638.png)

That is why I changed my approach and remove those two colors in my last commit and inserted the icon as a drawable png resource

Reply:
Can we use a `VectorDrawable` instead please so that this scales nicely on all screens?

Reply:
@nickbutcher I used to have it as a Vector asset, but I was placing the background with a Surface component, but I wasn't sure where to create the color because of this conversation. Else I could also create an image asset which would allow me to set the background there instead of using a surface component . What do you think?

Reply:
Oh I thought the "background color" was that red, I didn't see "icon". Yeah this should be a vector asset and the colors should be hardcoded in it, not in compose.

Reply:
I just added the vector with the hardcoded colors.